docs/elements/dispatcher.html

```diff
@@ -0,0 +1,99 @@
+<article-header>Dispatcher</article-header>
+
+<article-paragraphs>
+    <p>
+        Use <br>
+        <code class="underline">Blumilk\BLT\Features\Dispatcher</code> context <br>
+        or add <br>
+        <code class="underline">Blumilk\BLT\Features\Traits\Dispatcher</code> trait to test your application's
+        dispatcher features.
+    </p>
+</article-paragraphs>
+
+<section-header>Dispatcher Methods</section-header>
+<article-paragraphs>
+    <p>
+        The <code>Dispatcher</code> trait provides several methods to handle the dispatching of jobs and events within
+        your Laravel application. These methods ensure that your application's event and job dispatching features are
+        functioning as expected.
+    </p>
+</article-paragraphs>
+
+<article-paragraphs>
+    <p>
+        This snippet starts the bus fake, allowing you to simulate the dispatching of jobs within your tests. This is
+        useful for scenarios where you need to ensure that certain jobs are dispatched without actually executing them.
+    </p>
+</article-paragraphs>
+<code-snippet
+        gherkin="Given bus is running"
+        php="
+    /**
+     * @Given bus is running
+     * @throws BindingResolutionException
+     */
+    public function runBus(): void"></code-snippet>
+
+<article-paragraphs>
+    <p>
+        The <code>fakeEvents</code> method starts the event fake, allowing you to simulate the dispatching of events
+        within your tests. This is useful for scenarios where you need to ensure that certain events are dispatched
```

Comment:
A bit repetition about this "...are dispatched without actually executing them." Maybe just one time on the beginning will be enough?

Reply:
Okey, removed repeated texts